client/components/map/mapToolTip.jsx

```diff
@@ -1,12 +1,15 @@
 import React from 'react'
 import { Popup } from 'react-leaflet'
-const Tooltip = ({distance, trailName, ...props}) => (
+const Tooltip = ({distance, trailName, isActive, ...props}) => (
   <Popup {...props}>
     <span className='trail__tooltip'>
       <h5>
         { trailName || 'No name given' }
       </h5>
-      <p>Estimated Distance: {guessLength(props.popupContainer, distance.addLength)} Miles
+      <p>Estimated Distance: {isActive.b 
```

Comment:
Correct. That is why it is a [temporary](https://github.com/NathanBland/trails/pull/11/commits/20a811d8dbe4f6adea09fd1f2b73c3b44e86d17a) hack.